### DISCUSSION TITLE
No connection to the inverter

### DISCUSSION THREAD

**Discussion by flimon31** [tbnobody/OpenDTU#220] [Q&A] [answered]
Hello. I have a question. How to check the id of an inverter. ?? I have it 2 years and I do not know if I entered the correct number. In OpenDTU cannot see the connection to the inverter.

**Comment by ce3rd** [comment]
there is a sticker on the backside of the inverter - or maybe you can find the serial number on the bill...

**Reply by flimon31** [reply]
Sticker is unreadable after 2 y. I take number from the bill. I have no connection. How  to check problems ??. OpenDTU is in latest version. On system page I have connection with chip and model.

**Reply by ce3rd** [reply]
if you are sure, you have an opendtu-supported model (see readme)- then it could be something like in this issue -> https://github.com/tbnobody/OpenDTU/issues/199

**Reply by tbnobody** [reply]
If #199 would be the problem the inverter admin view would show "Unknown" as model. If the right model appears the serial is detected correctly.

**Reply by flimon31** [reply]
i have 1061xxx and yes this is MI-1500.

**Comment by tbnobody** [comment, marked as answer]
First of all, the only way to communicate with the inverter is the serial number. If you don't have the correct serial number than there is no way to find it out. What are the first 4 digits of the serial? If it's already 11xx then the inverter is at least in the range of supported models. (also #199 is already fixed an implemented correctly)

**Reply by flimon31** [reply]
1061

**Reply by tbnobody** [reply]
As you maybe saw yourself, MI- Inverters are unfortunatly not supported. They are using a different protocol

**Reply by flimon31** [reply]
Yes. Tx for   answers.
